app/data_providers/slate_providers/recommended_reads_slate_provider.py

```diff
@@ -0,0 +1,20 @@
+from app.data_providers.corpus.corpus_feature_group_client import CorpusFeatureGroupClient
+from app.models.corpus_recommendation_model import CorpusRecommendationModel
+from app.models.corpus_slate_model import CorpusSlateModel
+
+
+class RecommendedReadsSlateProvider:
```

Comment:
Nit: There is some boilerplate here causing some of the expressive ness of this class to be lost. As we think about future efforts we will probably want to consider creating a base class or other pattern for corpus feature group client slate models...

Reply:
Agreed.